recipe/meta.yaml

```
@@ -1,51 +1,92 @@
 {% set name = "pycytominer" %}
-{% set version = "0.2.0" %}
+{% set version = "1.0.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pycytominer-{{ version }}.tar.gz
-  sha256: 3f64cc584b50c80bd2141aab9b3acd77dbd1b651f3764b0d2e03bc963484f1bf
+  sha256: af18328fa484ca6a0413c70c0664b6503f117a79a9052c52631f52194bb0a97d
 build:
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv
   number: 0
 
 requirements:
   host:
+    - python >=3.8,<4.0
     - pip
-    - python >=3.4
+    - poetry
+    - poetry-dynamic-versioning
   run:
-    - numpy
-    - pandas
-    - python >=3.4
-    - scikit-learn
-    - sqlalchemy
-  
+    - python >=3.8,<4.0
+    - numpy >=1.16.5
+    - scipy >=1.5
+    - pandas >=1.2.0
+    - scikit-learn >=0.21.2
+    - sqlalchemy >=1.3.6,<2
+    - pyarrow >=8.0.0
+
 test:
+  requires:
+    - pip
   imports:
     - pycytominer
   commands:
     - pip check
-  requires:
-    - pip
+
 outputs:
   - name: pycytominer
     version: {{ version }}
   - name: pycytominer.collate
     version: {{ version }}
+    build:
+      noarch: python
+    requirements:
+      host:
+        - python >=3.8,<4.0
+      run:
+        - python >=3.8,<4.0
+        - {{ pin_subpackage('pycytominer', exact=True) }}
+        - cytominer_database >=0.3.4
+    test:
+      requires:
+        - pip
+      imports:
+        - pycytominer
+        - cytominer_database
+      commands:
+        - pip check
+  - name: pycytominer.cell-locations
+    version: {{ version }}
+    build:
+      noarch: python
     requirements:
-      - {{ pin_subpackage('pycytominer', exact=True) }}
-      - cytominer_database >=0.3.4      
+      host:
+        - fire >=0.5.0
+      run:
+        - python >=3.8,<4.0
+        - {{ pin_subpackage('pycytominer', exact=True) }}
+        - fsspec >=2023.1.0
+        - s3fs >=2023.4.0
+        - boto3 >=1.26.79
+        - fire >=0.5.0
+    test:
+      requires:
+        - pip
+      imports:
+        - pycytominer
+        - boto3
+      commands:
+        - pip check
```

```suggestion
      imports:
        - pycytominer.cyto_utils.cell_locations
```
probably better to just test an import from the package itself.

You already called "pip check" in the main test, so no need to repeat it here.